DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc

```diff
@@ -23,93 +23,101 @@
 #include "DQM/SiStripCommon/interface/SiStripHistoId.h"
 #include "TMath.h"
 
-SiStripMonitorTrack::SiStripMonitorTrack(const edm::ParameterSet& conf):
-  conf_(conf),
-  tracksCollection_in_EventTree(true),
-  firstEvent(-1),
-  genTriggerEventFlag_(new GenericTriggerEventFlag(conf.getParameter<edm::ParameterSet>("genericTriggerEventPSet"), consumesCollector(), *this))
-{
-  Cluster_src_   = conf.getParameter<edm::InputTag>("Cluster_src");
-  Mod_On_        = conf.getParameter<bool>("Mod_On");
-  Trend_On_      = conf.getParameter<bool>("Trend_On");
+
+#include "RecoLocalTracker/ClusterParameterEstimator/interface/StripClusterParameterEstimator.h"
+#include "RecoLocalTracker/Records/interface/TkStripCPERecord.h"
+
+SiStripMonitorTrack::SiStripMonitorTrack(const edm::ParameterSet& conf)
+    : conf_(conf),
+      tracksCollection_in_EventTree(true),
+      firstEvent(-1),
+      genTriggerEventFlag_(new GenericTriggerEventFlag(
+          conf.getParameter<edm::ParameterSet>("genericTriggerEventPSet"), consumesCollector(), *this)) {
```

Comment:
Hi @jandrea  I believe this could be causing the crash in the 3 workflows, since module SiStripMonitorTrack:HLTSiStripMonitorTrack is complaining, can you please check?
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0447f2/447/runTheMatrix-results/136.85_RunEGamma2018A+RunEGamma2018A+HLTDR2_2018+RECODR2_2018reHLT_skimEGamma_Offline_L1TEgDQM+HARVEST2018_L1TEgDQM/step3_RunEGamma2018A+RunEGamma2018A+HLTDR2_2018+RECODR2_2018reHLT_skimEGamma_Offline_L1TEgDQM+HARVEST2018_L1TEgDQM.log